object_detection/BoundingBox[] boxes